xml/Microsoft.Extensions.Options/OptionsFactory`1.xml

```diff
@@ -157,7 +157,7 @@
       </Parameters>
       <Docs>
         <param name="name">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Creates a new instance of options type</summary>
```

Comment:
Perhaps:

`Creates a new instance of options type` -> `Creates a new instance of type TOptions`

Reply:
Actually, this one needs to get more detail:

```suggestion
        <summary>Creates a new instance of a <typeparamref name="TOptions" /> object using the specified <paramref name="name" />.</summary>
```